cep-XXXX.md

```diff
@@ -0,0 +1,424 @@
+- *hashed `<component>`*: A string component of an OCI repository name or OCI tag, possibly the entire `<name>` or `<tag>`,
+  hashed with SHA256 and then formatted as `h<SHA256 in hexadecimal>`.
+
+### Specification Version and Compatible OCI Specifications
+
+This specification is labeled `v1`. Any implementation of this specification MUST use the `v1.*` OCI specification.
+
+### Example Forms of a Valid OCI conda URL
+
+The following are valid example forms of an OCI conda URL:
+
+```text
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]/<OCI-compatible subdir>/<OCI-encoded package name>:<OCI-encoded version>-<OCI-encoded build>
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]/<OCI-compatible subdir>/repodata.json:latest
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]/<OCI-compatible subdir>/repodata_from_packages.json:latest
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]/<OCI-compatible subdir>/run_exports.json:latest
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]/<OCI-compatible subdir>/patch_instructions.json:latest
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]/channeldata.json:latest
+```
+
+The various parts of the URL are defined below.
+
+### Channel Base URL and Subdirs
+
+The base URL for an OCI conda channel MUST be formatted as follows:
+
+```text
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]
+```
+
+where `<authority>` is any valid [RFC 3986][rfc3986] authority which is also supported by the OCI `v1.*` specification.
+This string will typically be `<hostname>[:<port>]` (e.g., `ghcr.io` with no port specified). OCI conda channel URLs MUST
+use `oci` as the scheme. OCI-compatible channel path and labels are defined below. If the `<OCI-encoded label>` is `main`,
+the entire optional label component MUST be omitted. Otherwise, the label component MUST be present.
+
+The base URL for an OCI conda channel is the canonical, unique identifier for the channel itself and SHOULD be used in
+conda ecosystem tools to refer to the channel.
+
+Conda `<OCI-compatible subdir>`s are appended to this base URL as follows:
+
+```text
+<OCI conda channel base URL>/<OCI-compatible subdir>
+```
+
+where `<OCI-compatible subdir>` is defined below.
+
+The following regexes MUST be satisfied by the `<OCI-compatible channel path>`, `<OCI-compatible subdir>`, and `<OCI-compatible label>`:
+
+- `<OCI-compatible channel path>`: `^[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*(\/[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*)*$` (same as the regex for an OCI repository `<name>`)
+- `<OCI-compatible label>`: `^[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*(\/[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*)*$` (same as the regex for an OCI repository `<name>`)
+- `<OCI-compatible subdir>`: `^[a-z0-9]+(-[a-z0-9]+)*$`
```

Comment:
This is the same as the one proposed in https://github.com/conda/ceps/pull/116, right? We should update this part to refer to that one once accepted.